HHS/OPRE-OPS
# OPS-966_Wire_Subit_Approval_Button

## What changed

- minor cleanup on API code (remove duplicate code)
- Added "In Review" styling to PreviewTable
- Added handleSubmitClick on Approve Agreement page
  - handles minor error checking; if no BLI are in DRAFT, button is disabled.
  - redirects back to `/agreements/` when complete
## Issue

#966

## How to test

Try to submit an Agreement that contains BLI with DRAFT status. Status of BLI should be updated to "In Review", should redirect to `/agreements/` list page.

## Screenshots

![image](https://github.com/HHS/OPRE-OPS/assets/56687505/884f3129-2f99-4dbf-956f-0257bde04901)
